### DISCUSSION TITLE
Z-wave 800LR Silabs relance z-wave - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#37] [Comments]
# Z-wave 800LR Silabs relance z-wave - Haade.fr

Depuis le rachat de la fabrication des puces Z-wave, et l’avancée significative sur le marché, Silabs s’apprête à rescussiter le Z-wave sur le marché Européen

https://haade.fr/fr/blog/z-wave-800-long-range-silabs-renouveau

**Comment by remydev** [comment]
Quel intéret de poursuivre avec Z-wave alors que theard est prévu très rapidement ?
D'autant, que je comprends que theard est finalement du zigbee modifié, il semble de mon point de vue que le Z-wave est dépassé.

**Reply by haade-administrator** [reply]
thread est un bon protocole, mais matter qui se rajoute en surcouche pour l'instant je ne vois pas l'intérêt ça revient à faire du wifi à cheval sur du maillage ( en l'occurence du zigbee ). Je pensais aussi z-wave mort et enterré mais Silabs le relance et le binding ( modules sont compatibles et retro-compatible entre eux ) est le point noir des autres protocoles. La distance de fonctionnement est un gros avantages surtout pour les grosses infrastructures.

Il y a de ça 8-10 ans tu trouvais à foison des modules z-wave comme c'est le cas pour zigbee actuellement, et je remarque que les nouveaux fabriquants comme zooz emboîtent le pas pour être lenouveau référent z-wave.
